examples/linearAerodynamics/linearAerodynamics.m

```diff
@@ -62,9 +65,10 @@
 %md The node has not cross section geometry to assign (an empty array is automatically set). Since the frame element has no implemented a hollow cylindrical cross section, then a `'generic'` cross-section dimensions in $y$ and $z$ directions is used. Thus the _elemCrossSecParams_ field is:
```

Comment:
```suggestion
%md The node type has no cross-section geometry to assign (an empty array is automatically set). Since the frame element has no implemented a hollow cylindrical cross-section, then a `'generic'` cross-section (in $y$ and $z$) is used. Thus the _elemCrossSecParams_ field is:
```